app/services/force/soql/preference_service.rb

```diff
@@ -7,15 +7,44 @@ class PreferenceService < Force::Base
       FIELD_NAME = :preferences
       FIELDS = load_fields(FIELD_NAME).freeze
 
-      def application_preferences(application_id)
+      def application_preferences_for_application(application_id)
         result = parsed_index_query(%(
-          SELECT #{query_fields(:show_preference)}
+          SELECT #{query_fields(:app_preferences_for_application)}
           FROM Application_Preference__c
           WHERE Application__c = '#{application_id}'
         ), :show_preference)
 
         result.map { |r| Force::Preference.from_salesforce(r).to_domain }
       end
+
+      def application_preferences_for_listing(opts)
```

Comment:
this file has some variable names with application abbreviated, some with preference abbreviated, and some with both.

I think we should settle on one or the other and stay as consistent as possible

Reply:
made it consistent :+1: